src/Game/Scripting/Commands.cs

```diff
@@ -24,845 +24,1069 @@
+            //Interpreter.RegisterCommandHandler("equipwand", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("buy", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("sell", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("clearbuy", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("clearsell", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("organizer", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("counter", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("replygump", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("closegump", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("cleartargetqueue", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargetlast", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargetself", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargetobject", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargettype", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargettile", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargettileoffset", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargettilerelative", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargetghost", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("autotargetground", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("cancelautotarget", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("getenemy", UnimplementedCommand);
+            //Interpreter.RegisterCommandHandler("getfriend", UnimplementedCommand);
+        }
 
-            if (item != null)
-            {
-                GameActions.DoubleClick(item);
-            }
-            else if (!quiet)
+        public static bool FindObject(CommandExecution execution)
+        {
+            var serial = execution.Params.NextAs<uint>(ParameterList.Expectation.Mandatory);
+            var color = execution.Params.NextAs<ushort>();
+            var source = execution.Params.NextAs<string>();
+            var amount = execution.Params.NextAs<int>();
+            var range = execution.Params.NextAs<int>();
+
+            Entity entity = CmdFindEntityBySerial(serial, color, source, range);
+            if (entity != null)
             {
-                GameActions.Print($"Script Error: Couldn't find '{graphicString}'", type: MessageType.System);
+                if ((entity is Item item && item.Amount > amount) || entity is Mobile)
+                    Aliases.Write<uint>("found", entity.Serial); // should this be a scope variable?
```

Comment:
In steam, 'found' is an alias that gets updated by various functions. Basically like what you have here.

The original steam doesn't have "scopes" but I do think they could be leveraged as a major improvement. And I think we may later want to reconsider whether 'found' remains an alias or becomes a scoped variable. But for now, it's an alias (it even persists between scripts).